src/components/Transaction.vue

```diff
@@ -98,133 +93,109 @@
   </FrameOutside>
 </template>
 
-<script>
-import FrameOutside from '@/components/FrameOutside.vue'
-import TransactionDetails from '@/components/TransactionDetails.vue'
-import InputsOutputs from '@/components/InputsOutputs.vue'
-import Amount from '@/components/Amount.vue'
-import Address from '@/components/Address.vue'
+<script setup lang="ts">
+import { ref, computed, type Ref } from 'vue'
+import PositiveIcon from '@/resources/svg/positive.svg'
+import NegativeIcon from '@/resources/svg/negative.svg'
+
+const showDetails: Ref<boolean> = ref(false)
 
-export default {
-  name: 'Transaction',
-  components: {
-    Address,
-    Amount,
-    FrameOutside,
-    InputsOutputs,
-    TransactionDetails,
+const props = defineProps({
```

Comment:
Now that we are refactoring this, I´d modify the numbers of props this component expects. I'd add a new prop `transaction`